Assets/Scripts/Button Cell.cs

```diff
@@ -0,0 +1,50 @@
+
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+public class ButtonCell : Cell
+{
+    [SerializeField] private TextMeshProUGUI cellText;
+    [SerializeField] private Image image;
+    private System.Action callback;
+    private ButtonCellData bData = new();
+
+    public override void Configure(CellData data)
+    {
+        bData = data as ButtonCellData;
+        callback = bData.action;
+        SetTextValue(bData.text);
+        if (bData.isImageActive)
```

Comment:
this isnt something Id expect you to know, but specifically naming variables with s, i, or b should be carefully considered. There is an old naming convention where boolean variables are prefixed with be like "bIsPressed". This is no longer considered good practise, but people (me) will sometimes get confused thinking this is a bool. In general if in doubt write the full name, like buttonData

on the other hand this is an instance where you probably wanted the member variable of the class to be just called "data", which conflicts with the variable name passed into this function. This is a bit contentious among programmers but I personally dont like renaming variables from what the "should" be to avoid conflicts. 

I prefer naming both data, then using this.data = data to differentiate between the member variable (this.data) from the locally defined variable (data) - up to you!